src/components/Thought.js

```diff
@@ -151,6 +153,18 @@ export const Thought = connect(({ cursor, cursorBeforeEdit, expanded, expandedCo
 
       const { thoughtsRanked: thoughtsFrom } = monitor.getItem()
       const thoughtsTo = props.thoughtsRankedLive
+      const isRootOrEM = isRoot(thoughtsFrom) || isEM(thoughtsFrom)
+      const oldContext = rootedContextOf(thoughtsFrom)
+      const newContext = rootedContextOf(thoughtsTo)
+      const sameContext = equalArrays(oldContext, newContext)
```

Comment:
Could you explain the use of `sameContext` here? I was thinking that we should error just on `isRootOrEM`, but let me know if I'm missing something.

Reply:
`sameContext ` was used to determine if the ROOT_TOKEN/EM_TOKEN is not being dropped into another context, i.e. dropped as a sub-thought.

Reply:
Okay, I understand now. Thanks!